test/units/testsuite-70.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+set -ex
+set -o pipefail
+
+export SYSTEMD_LOG_LEVEL=debug
+
+# Prepare fresh disk image
+dd if=/dev/zero of=/test.img bs=1024k count=100 status=none
+echo -n passphrase >/tmp/passphrase
+cryptsetup luksFormat -q --use-urandom test.img /tmp/passphrase
+
+# Enroll unlock with default PCR policy
+env PASSWORD=passphrase systemd-cryptenroll --tpm2-device=auto /test.img
+/usr/lib/systemd/systemd-cryptsetup attach test-volume /test.img - tpm2-device=auto,headless=1
+cryptsetup luksClose test-volume
+
+# Check with wrong PCR
+tpm2_pcrextend 7:sha256=0000000000000000000000000000000000000000000000000000000000000000
+/usr/lib/systemd/systemd-cryptsetup attach test-volume /test.img - tpm2-device=auto,headless=1 && exit 1
```

Comment:
When expecting a failure we use the pattern:

`foo && { echo 'unexpected success'; exit 1; }`